### ISSUE TITLE
git-status for empty directory is misleading

### ISSUE THREAD

**Issue by hoganbc** [athityakumar/colorls#363]
### Description

When I view a _directory_ that does not include a git-tracked file, it has the ✓, indicating tracked and unmodified.

Recreate
```bash
mkdir mvp-status
cd mvp-status
git init
Initialized empty Git repository in /Users/XXX/code/mvp-status/.git/

mkdir untracked-dir
colorls --git-status                                                                                                                                                                                                     
     ✓    untracked-dir/
```
![Screen Shot 2020-05-27 at 22 22 05](https://user-images.githubusercontent.com/32182119/83091135-85cbf280-a068-11ea-9e8d-2b24cfe30025.png)

`untracked-dir`does not have a file in it, let alone one that git is aware of. I expect it to not have an icon (~aligns with how git sees the directory~) or have the question marks for untracked.

When I add a file it behaves as expected, with the ? status.

``` bash
touch untracked-dir/README.md
colorls --git-status
     ?    untracked-dir/
```
![Screen Shot 2020-05-27 at 22 26 34](https://user-images.githubusercontent.com/32182119/83091478-47830300-a069-11ea-8bcc-7a4ecf9fa493.png)

- Relevant Issues : (none)
- Relevant PRs : (none)
- Type of issue :
  - [ ] Installation
  - [ ] Font-related
  - [ ] Feature request
  - [x] Bug in existing feature
  - [ ] Developer mode : Code quality / Tests / Documentation

**Comment by hoganbc** [comment]
Looking through `lib/colorls/git.rb`, I think it runs the following command to get the "mode". I replaced `repo_dir` with the path to the example directory.

```bash
git status untracked-dir --porcelain -z -unormal --ignored .
?? untracked-dir/%
```

This is the status I was expecting. I haven't yet found the spot where the mode is swapped from `?` to `✓`.

**Comment by hoganbc** [comment]
Ah once I removed `untracked/README.md` the result from the git command was empty. So the ["if modes empty" clause](https://github.com/athityakumar/colorls/blob/b2def8eea9506727acb25b07fd31bec0d5eb863d/lib/colorls/git.rb#L34) is executed.

I realize that this is the exact same behavior that git status has for an unmodified file. The difference is that git doesn't show anything for an unchanged tracked file, but this flag shows a "✓".

I didn't find a discussion, commit description, or comment on PR #299 that seems to introduce this logic.

Is "✓" the best option for untracked directories and unmodified files?

**Comment by hoganbc** [comment]
What if `git_subdir_status` checks if the directory is empty when the status is `''`. If it's empty than mode is `'?'`.

**Comment by avdv** [comment]
You are probably using an old version. This has been fixed in 1.3.2, see fb39a9da6dcdff5430cfdfb01394020aaa2f8960

**Comment by hoganbc** [comment]
Yup. I was using 1.2.0. Upgraded to 1.4.1 and it behaves as I'd expect and as the commit indicates. Thanks!
